demosplan/DemosPlanCoreBundle/Permissions/Permissions.php

```diff
@@ -145,9 +147,34 @@ public function initPermissions(UserInterface $user): PermissionsInterface
         // set Permissions which are dependent on role but independent of procedure
         $this->setGlobalPermissions();
 
+        // set Permissions which are dependent on orga
+        $this->setOrgaPermissions();
```

Comment:
couldn't this be more generic like `loadDynamicPermissions`?

Reply:
Updated in new version

Reply:
done here https://github.com/demos-europe/demosplan-core/commit/219857ca07df9886f4701178c08933285e6b2b9f